README.md

```diff
@@ -20,8 +21,8 @@ To get a local copy up and running follow these simple example steps.
 <h3 dir="auto"><a id="user-content-prerequisites" class="anchor" aria-hidden="true" href="#prerequisites"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Author</h3>
 <h2 dir="auto"><a id="user-content-built-with" class="anchor" aria-hidden="true" href="#built-with"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>👤 Author1</h2>
 <ul>
-  <li>GitHub:<a href="https://github.com/vaderonline">vaderonline!  &nbsp; Hire !</a></li>
-  <li>Linkedin:<a href="https://www.linkedin.com/in/vishal-kumar-514455152/">Vishal kumar &nbsp; Hire !</a></li>
+  <li>GitHub:<a href="https://github.com/vaderonline">vaderonline!  &nbsp; <mark>contact me </mark></a></li>
+  <li>Linkedin:<a href="https://www.linkedin.com/in/vishal-kumar-514455152/">Vishal kumar &nbsp;<mark> Contact me</mark</a></li>
   </ul>
```

Comment:
- [ ] Kindly fill all missing information and correct typo errors in your Readme file. you can follow this [template](https://github.com/microverseinc/readme-template/blob/master/README.md)
- [ ] Make sure all your sections have the proper background image (decorations are always backgrounded images, not img tags)

Reply:
Thanks for reviewing my project as you mention about  the background image I used the same background image as the template have [figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)